src/NDItk/depletion/ReactionMultiplicities/src/getPostFix.hpp

```diff
@@ -0,0 +1,8 @@
+static std::string getPostFix( const ReactionMultiplicityType& type ) {
+
+  return type == ReactionMultiplicityType::All
```

Comment:
What happens if there isn't a postfix or if it isn't all, few, or rmo?

Reply:
This has been removed in part10